packages/common/src/components/Footer.jsx

```diff
@@ -7,23 +7,38 @@
  */
 
 import React from 'react'
+import { FooterLink } from '@kernel/common'
 
 const defaults = {
-  backgroundColor: 'bg-gray-900',
-  textColor: 'text-white'
+  backgroundColor: 'bg-kernel-dark',
+  textColor: 'text-white',
+  footerLinks: [
+    { app: 'explorer', title: 'explorer' },
+    { app: 'unprofile', title: 'unprofile' },
+    { app: 'adventures', title: 'adventures' },
+    { app: 'wallet', title: 'portal' },
+  ]
 }
 
-export default function Footer (props) {
-  const { children } = props
-  const backgroundColor = props.backgroundColor || defaults.backgroundColor
-  const textColor = props.textColor || defaults.textColor
-
+export default function Footer({ children, backgroundColor = defaults.backgroundColor, textColor = defaults.textColor, footerLinks = defaults.footerLinks }) {
   return (
     <>
       <footer className={backgroundColor}>
-        <div className={`px-4 py-6 text-sm ${textColor} font-semibold`}>
+        <div className={`px-4 py-6 text-sm ${textColor}`}>
           <div className='text-center'>
             {children}
+            <ul className='flex justify-center flex-col lg:flex-row list-none ml-3 mt-3 lg:mt-0 font-semibold'>
+              {!children && footerLinks.map((link, idx) => {
```

Comment:
I'd put a wrapper div around both the `ul` and the div below it (the "built at KERNEL" div), and make that whole thing dependent on `!children`, instead of the `li`s. That way when you pass `children`, it won't render the empty `ul` or the "built at KERNEL".

Reply:
Yes! You're right!
Added an empty wrapper as you suggested and moved the condition one notch up. Simplified everything even further.
Thanks!